src/main/scala/com/campudus/tableaux/router/auth/permission/Condition.scala

```diff
@@ -111,7 +111,7 @@ case class ConditionLangtag(jsonObject: JsonObject) extends ConditionOption(json
     // At this point, the value for the column type must already have been checked. -> checkValueTypeForColumn
     objects.columnOpt match {
       case Some(column) =>
-        if (column.languageType == MultiLanguage) {
+        if (column.languageType == MultiLanguage || column.languageType.toString == LanguageType.COUNTRY) {
```

Comment:
Instead of the String equals check (`column.languageType.toString == LanguageType.COUNTRY`), can't we do the same as with MultiLanguage e.g.
`column.languageType == MultiCountry`

Reply:
Unfortunately not with if, because `MultiLanguage` is a case object and `MultiCountry` is a case class. Pattern matching could work, I'll try.